src/Symfony/Component/JsonPath/Tests/Test/JsonPathAssertionsTraitTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Symfony\Component\JsonPath\Tests\Test;
+
+use PHPUnit\Framework\AssertionFailedError;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\JsonPath\Test\JsonPathAssertionsTrait;
+
+class JsonPathAssertionsTraitTest extends TestCase
+{
+    use JsonPathAssertionsTrait;
+
+    public function testAssertJsonPathEqualsOk()
+    {
+        self::assertJsonPathEquals([1], '$.a[2]', self::getSimpleCollectionCrawlerData());
+    }
+
+    public function testAssertJsonPathEqualsOkWithTypeCoercion()
+    {
+        self::assertJsonPathEquals(['1'], '$.a[2]', self::getSimpleCollectionCrawlerData());
+    }
+
+    public function testAssertJsonPathEqualsKo()
+    {
+        try {
+            self::assertJsonPathEquals([2], '$.a[2]', self::getSimpleCollectionCrawlerData());
+
+            $this->fail();
+        } catch (AssertionFailedError $exception) {
+            self::assertSame('Failed asserting that two arrays are equal.', $exception->getMessage());
+        }
+    }
+
+    public function testAssertJsonPathNotEqualsOk()
+    {
+        self::assertJsonPathNotEquals([2], '$.a[2]', self::getSimpleCollectionCrawlerData());
+    }
+
+    public function testAssertJsonPathNotEqualsKo()
+    {
+        try {
+            self::assertJsonPathNotEquals([1], '$.a[2]', self::getSimpleCollectionCrawlerData());
+
+            $this->fail();
+        } catch (AssertionFailedError $exception) {
+            self::assertMatchesRegularExpression('/^Failed asserting that .+ is not equal to .+\.$/s', $exception->getMessage());
+        }
+    }
+
+    public function testAssertJsonPathCountOk()
+    {
+        self::assertJsonPathCount(6, '$.a[*]', self::getSimpleCollectionCrawlerData());
+    }
+
+    public function testAssertJsonPathCountOkWithFilter()
+    {
+        self::assertJsonPathCount(2, '$.book[?(@.price > 25)]', <<<JSON
+{
+    "book": [
+        { "price":  10 },
+        { "price":  20 },
+        { "price":  30 },
+        { "price":  40 }
+    ]
+}
+JSON
+        );
```

Comment:
It may be a personal preference, but why not indent the HEREDOC?

```suggestion
        self::assertJsonPathCount(2, '$.book[?(@.price > 25)]', <<<'JSON'
            {
                "book": [
                    { "price":  10 },
                    { "price":  20 },
                    { "price":  30 },
                    { "price":  40 }
                ]
            }
            JSON);
```

Reply:
Still not used to the fact that we can now indent it, but I agree 🙂